content/docs/handling-events.md

```diff
@@ -29,27 +29,31 @@ React میں یوں لکھا جائے گا
 </button>
 ```
 
+<<<<<<< HEAD
 React  میں Events کا پہلے سے طے شدہ ردعمل بدلنے کے لئے return `false` کرنا کافی نہیں، آپ کو واضح طور پر `preventDefault` کا استعمال کرنا پڑے گا ۔ مثال کے طور پر ایک HTML link کا طے شدہ ردعمل ایک نیا صفحہ کھولتا ہے، جس کو آپ کچھ یوں تبدیل کر سکتے ہیں
+=======
+Another difference is that you cannot return `false` to prevent default behavior in React. You must call `preventDefault` explicitly. For example, with plain HTML, to prevent the default form behavior of submitting, you can write:
```

Comment:
This paragraph has been slightly changed. Here is the diff between previously translated and current version
![image](https://user-images.githubusercontent.com/3725386/120371080-c373f300-c32e-11eb-9ed7-9d3a7eb5ca7d.png)

Reply:
I think it is more like a rebase to base project so perhaps we take it in and then update it in a separate issue?

Reply:
@jabranr done. I have resolved this using upstream changes.